### ISSUE TITLE
The avg metric is  wrong

### ISSUE THREAD

**Issue by Hoainam25699** [webdacjs/simple-pivot-table#24]
In the source code , the avg function is wrong so the result is wrong .
I rewritet it 

```
  if (aggregator === 'avg') {
  
    var _rawValue = values.reduce(function (a, b) {
      return a + (getNumericValue(b) || 0);
  
    }, 0) / values.length;
    return Math.round( _rawValue);
  }
```

**Comment by webdacjs** [comment]
Thanks for your report. I have test for this function (please check https://github.com/webdacjs/simple-pivot-table/blob/main/src/components/utils/getAggregated.test.js#L29. can you further clarify why this is wrong with an example? Thank you.

**Comment by Hoainam25699** [comment]
I'm glad you checked.
in the reduce function the parameter a is the cumulative sum, so in the case of b = 0 the function will be a + a = 2a.
for example [5, 10, 0] the expected result is 5 but the formula is  ( ( 5 + 10 ) + ( 0 || (5+10)  ) / 3  = 10.

**Comment by Hoainam25699** [comment]
In this your code 
```
 if (aggregator === 'avg') {
    var _rawValue = values.reduce(function (a, b) {
      return a + (getNumericValue(b) || a);
    }, 0) / values.length;


```
In the case of b = 0 the function will be a + a = 2a

**Comment by webdacjs** [comment]
Sorry I have been busy during the last couple of days. But thank you for your report I agree this can occur when the last value of the array is 0, I assign this to myself. I will publish new version soon and your fix will be there. Again thank you for taking the time to report the issue.

**Comment by webdacjs** [comment]
This has been fixed in the version 2.8.4. Closing this issue
